Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -32,7 +32,13 @@
 #import "RemoteLayerTreeHost.h"
 
 #if PLATFORM(VISION)
-#import "RealitySystemSupportSPI.h"
```

Comment:
You certainly don't have to delete the SPI header, it's orthogonal to the soft linking. But this is fine too (as a mostly-revert).